collectors/auth0/utils.js

```diff
@@ -0,0 +1,40 @@
+function getAPILogs(auth0Client, state, accumulator, maxPagesPerInvocation) {
+    let pageCount = 0;
+
+    let params = state.last_log_id ? { from: state.last_log_id, take:100 } : { q: "date=[" + state.since + " TO *]", per_page:100, sort: "date:1" };
+    let nextLogId = state.last_log_id ? state.last_log_id : null;
+    let lastLogTs = null;
+
+    return new Promise(function (resolve, reject) {
+        getData(params);
+        function getData(params) {
+            if (pageCount < maxPagesPerInvocation) {
```

Comment:
Do we need to guard against page counts over 10? The auth0 api has a max of ten pages right?

Reply:
we are not using auth0 pagination, we created our custom pagination and collector can pull more than 10 pages, It's tested

Reply:
Sounds good.